### PR TITLE
Madrid meeting rooms booking guideline

### PR DESCRIPTION
**Pull request by gretacv**
This guideline aims at giving information on the available Google Calendars for both meeting rooms in Madrid.

### CODE REVIEW

File: guidelines/madrid-meeting-room.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+These steps to subscribe to a calendar via the Google Calendar web should only be done the first time: 
+
+ 1. on the left panel next to `Other calendars` click on the `+` sign
+ 2. select `Subscribe to calendar`
+ 3. search for `meeting`
+
+Repeat to subscribe to both calendars. 
+
+# Book a meeting room
+Via the Google Calendar web check the availability of the meeting rooms by enabling their calendars. Drag in the calendar the desired slot. In the pop up select the calendar of the room of interest. From here it is also possible to invite all the people involved in the meeting.
```
Comment (dhakelila) [inline guidelines/madrid-meeting-room.md]: What I do is to create the event on my calendar so I'm the owner and then, I invite the meeting room. 
But I guess both ways work as far as the slot gets book in the correct calendar.

Comment (dhakelila) [inline guidelines/madrid-meeting-room.md]: Maybe worth to add a note about being sure you are booking the correct room.

Comment (dhakelila) [inline guidelines/madrid-meeting-room.md]: Should we make explicit that by booking the room you are also booking the material inside? Like TV set and owl?

Comment (gretacv) [reply guidelines/madrid-meeting-room.md]: Thanks. I have added this.

Comment (gretacv) [reply guidelines/madrid-meeting-room.md]: Thanks. I have added this. Do you think it should be reminded somewhere else in the guideline?

Comment (gretacv) [reply guidelines/madrid-meeting-room.md]: To me it makes sense that both the TV set and the Owl belong to the Meeting Room. I am thinking that then we should also put that the Jabra and the little web cam live on the forest table. What do you think?

Comment (dhakelila) [reply guidelines/madrid-meeting-room.md]: I think it's enough

Comment (dhakelila) [reply guidelines/madrid-meeting-room.md]: Agree. This way, also we might create the habit of keeping those (the little cam and one of the jabras) there

Comment (gretacv) [reply guidelines/madrid-meeting-room.md]: I have added a Material available section.

**Pull request merged**